Docker/Dockerfile

```diff
@@ -23,7 +23,16 @@ RUN \
   # && \
   #apt-get install -y python3-pip
 
-ENV JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-amd64
+# For MAC M1:
+# ENV JAVA_HOME=/usr/lib/jvm/java-8-openjdk-arm64
+
+RUN ARCH=`uname -m`
```

Comment:
Have you tested this? I recollect that environment variables may not pass from one RUN to another. You may need to inline the `RUN ARCH='uname -m'` in the next line...